docs/docker.rst

```diff
@@ -80,4 +80,18 @@ Secondary workers:
         bash -c "/usr/sbin/sshd -p 12345; sleep infinity"
 
 
+Adding Mellanox RDMA support
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If you have Mellanox NICs, we recommend that you ensure that Mellanox devices (``/dev/infiniband``) are mounted into the container
```

Comment:
should it be "mounted in"?

Reply:
Also, "we recommend that you mount your Mellanox devices (``/dev/infiniband``) in the container"